weblab/static/js/expt_common.js

```diff
@@ -284,6 +285,11 @@ function getKeyValues(file, numTraces)
  */
 function allowPlotExport(filename, datasets, axisLabels)
 {
+    try {
+        var isFileSaverSupported = !!new Blob;
+    } catch (e) {
+        return; // No saveAs() available
+    }
```

Comment:
Is this in case of the browser not supporting the FileSaver library?

Reply:
Yes, that's the check the FileSaver docs recommend.